File: FaceSample/app/src/main/java/com/regula/facesample/items/features/FaceSdkDeinitItem.java
package com.regula.facesample.items.features;

import android.content.Context;

import com.regula.facesample.items.CategoryItem;
import com.regula.facesdk.FaceSDK;

public class FaceSdkDeinitItem extends CategoryItem {

    @Override
    public void onItemSelected(Context context) {
        FaceSDK.Instance().deinit();
    }

    @Override
    public String getDescription() {
        return "Deinitialization FaceSDK resources";
    }

    @Override
    public String getTitle() {
        return "Deinitialization SDK";
    }
}

File: FaceSampleKotlin/app/src/main/java/com/regula/facesamplekotlin/items/features/FaceSdkDeinitItem.kt
package com.regula.facesamplekotlin.items.features

import android.content.Context
import com.regula.facesamplekotlin.category.CategoryItem
import com.regula.facesdk.FaceSDK

class FaceSdkDeinitItem : CategoryItem() {

    override fun onItemSelected(context: Context) {
        FaceSDK.Instance().deinit()
    }

    override val title: String
        get() = "Deinitialization SDK"

    override val description: String
        get() = "Deinitialization FaceSDK resources"
}